index.html

```diff
@@ -9,20 +9,22 @@
     <link rel="stylesheet" href="./style.css">
     <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+
 </head>
 
 <body>
     <!-- Header -->
     <section class="header">
         <header class="nav">
             <nav>
-                <img class="menu" src="./assets/ic_menu.svg" alt="hamburger menu">
-                <ul>
+                <img class="menu" id="hamburger" src="./assets/ic_menu.svg" alt="hamburger menu">
+                <ul id="ul">
                     <li><a class="underline" href="#Home">Home</a></li>
                     <li><a href="#project">Portfolio</a></li>
                     <li><a href="#about">About</a></li>
                     <li><a href="#contact">Contact</a></li>
                     <div class="mob-abs-2"></div>
+                    <img id="cross" class="menu-cross" src="./assets/cross.svg" alt="cross">
```

Comment:
- [ ] Your menu is great but it's just a tiny change you should position this box below as in the Figma design in order to match the Figma design 
- box 
<img width="135" alt="Screenshot 2022-11-14 at 15 11 34" src="https://user-images.githubusercontent.com/61790587/201681884-caba7e58-23b4-4697-8442-278425430484.png">

- Yours 
<img width="446" alt="Screenshot 2022-11-14 at 15 11 40" src="https://user-images.githubusercontent.com/61790587/201681806-4729a6dd-8a58-46f1-bc54-d2ea258872bf.png">
- Figma 
<img width="299" alt="Screenshot 2022-11-14 at 15 11 50" src="https://user-images.githubusercontent.com/61790587/201681840-f5dabf12-7449-4f00-b29e-db6cfadc4365.png">